tests/utils/test_ensembl_rest_clients.py

```diff
@@ -43,6 +43,23 @@ def test_ping_ensemble_38(ensembl_rest_client_38):
     assert data == {"ping": 1}
 
 
+@responses.activate
+def test_liftover(ensembl_rest_client_37, ensembl_liftover_reasponse):
+    """Test send request for coordinates liftover"""
+    # GIVEN a patched response from Ensembl
+    url = "http://grch37.rest.ensembl.org/map/human/GRCh37/X:1000000..1000100/GRCh38?content-type=application/json"
+    responses.add(
+        responses.GET,
+        url,
+        json=ensembl_liftover_reasponse,
```

Comment:
```suggestion
        json=ensembl_liftover_response,
```

Reply:
Thanks, at least I was consistent with that typo! 😄